src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -397,4 +400,18 @@ protected function getTemplateManager()
 
         return $this->templateManager;
     }
+
+    private function renderWithCspNonces(Request $request, $template, $variables, $code = 200, $headers = array('Content-Type' => 'text/html'))
```

Comment:
according to your talk at the SymfonyLive right now, wouldn't it be better to use signatures instead ?